.github/workflows/docker.yml

```diff
@@ -0,0 +1,98 @@
+name: Docker
+on:
+  push:
+    branches:
+      - main
+      - fixes1
+  release:
```

Comment:
Will this publish a production build (i.e. no debug)?

Reply:
Yes. `production.Dockerfile` is used below.